Models/reviews.model.js

```diff
@@ -17,3 +17,28 @@ exports.fetchReview = (id) => {
       return table.rows;
     });
 };
+
+exports.fetchAllReviews = () => {
+  const promisOne = db.query(
+    `SELECT owner,title,review_id,category,review_img_url,created_at,votes,designer FROM reviews ORDER BY created_at DESC `
+  );
+  const promiseTwo = db.query(`SELECT review_id FROM comments `);
+  return Promise.all([promisOne, promiseTwo])
+    .then((reviews) => {
+      for (let i = 0; i < reviews[0].rows.length; i++) {
+        reviews[0].rows[i].comment_count = 0;
+      }
+      for (let t = 0; t < reviews[0].rows.length; t++) {
+        for (let z = 0; z < reviews[1].rows.length; z++) {
+          if (reviews[1].rows[z].review_id === reviews[0].rows[t].review_id) {
+            reviews[0].rows[t].comment_count++;
+          }
+        }
+      }
+
+      return reviews[0].rows;
+    })
+    .then((arrOfReviews) => {
+      return arrOfReviews;
+    });
```

Comment:
There is a lot of javascript logic here to format the data, however this can and should all be done in a single sql query.